src/serviceWorker.js

```diff
@@ -121,7 +121,7 @@ function checkValidServiceWorker(swUrl, config) {
     })
     .catch(() => {
       console.log(
-        'No internet connection found. App is running in offline mode.'
+        'No internet connection found. app is running in offline mode.'
```

Comment:
Do you need this log

Reply:
This was pulled form another dev code, just asked me for styling fix.
As far as I know they needed it to test redux and ensure the correct error is used for debugging the redux implementation